frontend/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Delete this file maybe? Autogenerated right?

Reply:
I decided on leaving the default files in, in case people writing tests in future want mock data. Folder needs to have a file in it for git to recognise it. I'll remove the data from it though.